electrum/gui/qt/installwizard.py

```diff
@@ -670,8 +676,16 @@ def get_authenticator_pubkey(self, run_next, disallowed_key=None):
         label.setTextInteractionFlags(Qt.TextBrowserInteraction)
         label.setWordWrap(True)
 
+        if self.wallet_type == '2-key':
+            hint_url = 'https://translations.bitcoinvault.global/pdf/BTCV_Tutorial/BTCV-ExtendedTutorial-en.pdf#page=55'
+        if self.wallet_type == '3-key':
```

Comment:
Use `elif` here. The urls look like almost the same. What about using only page reference under if statement?

Reply:
done